daprdocs/content/en/operations/components/component-scopes.md

```diff
@@ -29,33 +34,64 @@ spec:
     value: redis-master:6379
 ```
 
-In this example, the Redis component is only accessible to Dapr instances running inside the `production` namespace.
-
-### Example of component namespacing in self-hosted mode
-
-In self hosted mode, a developer can specify the namespace to a Dapr instance by setting the `NAMESPACE` environment variable.
-If the `NAMESPACE` environment variable is set, Dapr will not load any component that does not specify the same namespace in its metadata.
-
-Considering the example above, to tell Dapr which namespace it is deployed to, set the environment variable:
+To tell Dapr which namespace it is deployed to, set the environment variable:
 
 MacOS/Linux:
 
 ```bash
-export NAMESPACE=production
-
+export NAMESPACE = production
```

Comment:
This does not work on Linux (requires no spaces between env var name and assigned value)